root/edit/details/AddRelationshipType.js

```diff
@@ -33,6 +34,7 @@ type AddRelationshipTypeEditT = {
     +long_link_phrase: string,
     +name: string,
     +orderable_direction?: number,
+    +relationship_type: LinkTypeT,
```

Comment:
Is there a case where `defined($self->entity_id)` isn't true? 'Cause we don't output a `relationship_type` key in that case.
 
```suggestion
    +relationship_type?: LinkTypeT,
```

Reply:
Oh, sorry, yes, I forgot changing this once I found out we don't have entity_id for old edits. Will amend.